extract_unique_sensors.py

```diff
@@ -10,11 +10,12 @@
         for candidate_line in line.split():
             candidate_lines.append(candidate_line)
 
-# Loop over candidate_lines and extract sensor names
+#  Loop over candidate_lines and extract sensor names
 sensor_names = []
 for sensor in candidate_lines:
-    # Typical output from "sensor" variable:
-    # kat.sensors.anc_api_version   kat.sensors.anc_ganglia_api_version   kat.sensors.anc_ganglia_kat_monctl_system_os_name
-    sensor = sensor.split('.')[-1].strip("\n")
+#  Typical output from "sensor" variable:
+#  kat.sensors.anc_api_version   kat.sensors.anc_ganglia_api_version   kat.sensors.anc_ganglia_kat_monctl_system_os_name
```

Comment:
wrt to spaces after `#`, please familiarize yourself with https://www.python.org/dev/peps/pep-0008/#id31

This is the sacred [style guide for Python](https://www.python.org/dev/peps/pep-0008), know it and breath it :laughing:

Reply:
@mamkhari